### ISSUE TITLE
Add crossfade 

### ISSUE THREAD

**Issue by Victor332lop** [fast4x/RiMusic#4986]
### Description

Se que está app es muy buena y todo, pero estaría bueno agregar un mezclador de música o crossfade, cómo se llame, pero estaría muy bueno, algo como el de Spotify 

### Suggested Solution

![Screenshot_20241220-081236_Spotify](https://github.com/user-attachments/assets/b2615629-32e0-49b4-8883-e8e2065b5e44)
La verdad estaría muy bueno 

### Additional Context

Saludos, les deseo un muy buen futuro a su linda app q hace a muchos felices 🥰

**Comment by Morello230** [comment]
I agree and really hope that RI MUSIC will have a Crossfade feature like the one on Spotify.
Especially when playing songs on a playlist.

Did the old version of RI MUSIC ever have this crossfade feature?
If so, what version is it?
